### DISCUSSION TITLE
Bilayers or Bilayer

### DISCUSSION THREAD

**Discussion by gnodar01** [bilayer-containers/bilayers#6] [Q&A] [answered]
I have just realized I have been interchangeably calling this project both "Bilayer" and "Bilayers". Also this org is called "bilayer-containers" and this repo is called "bilayers". My inclination is to call it "Bilayers" in the sense that each wrapped tool + interface is a single "bilayer", and we have many of them forming a collection of "bilayers". Other thoughts?

**Comment by rajavishah** [comment, marked as answer]
'Bilayers' also makes sense to me, as we are making it for multiple tools and interfaces. I've created Project with name 'Bilayers', later if we want we can change!

**Comment by bethac07** [comment]
Bilayers, for sure, that was just already taken as a GH org.
